chwifi

```diff
@@ -91,7 +91,7 @@ update_profiles() {
     profiles=""
     # Grab other profiles and concatenate in string, using comma as delimiter
     for entry in "/etc/netctl"/*; do
-        if [ -f "$entry" ]; then
+        if [ -f "$entry" ] && [[ ! "$entry" =~ work ]] ; then
```

Comment:
This is bash pattern-matching, which may be unreliable. Would this match a profile called `not-work`? If so, that's undesirable behaviour.

Reply:
You are absolutely right, I must have written it hastily. a430bd9 fixes this.

Reply:
I changed it a bit further. 3404c0e fixes this in `chwifi` by using config var. fa3359b and 8c5df5e  fixes this in `setup.sh` by prompting user for work profile name and using this as regex when getting profiles. Finally I updated the default value for work profile in the sample config, to ensure the profile name being regexed into the right spot.

Reply:
See latest comment on using `$work_profile`